README.md

```diff
@@ -37,7 +37,7 @@ to the users, which is useful for debugging and introspection.
 This also provide a pathway for JAX users to integrate JAX functions serialized
 via `jax2tf.convert`, back into their existing JAX codebases.
 
-See [section](#alternatives) at the end for comparison with an alternative
+See [section](#alternatives-jax2tfcall_tf) at the end for comparison with an alternative
```

Comment:
Should this not be #alternatives-jax2tfcall-tf ?

Reply:
I got it. Thank you for your response